docs-src/python/how-to-send-a-signal-from-a-workflow-in-python.md

```diff
@@ -0,0 +1,27 @@
+---
+id: how-to-send-a-signal-from-a-workflow-in-python
+title: How to Send a Signal from a Workflow in Python
+sidebar_label: Send Signal from Workflow
+description: Use `get_external_workflow_handle_for` to get a typed Workflow handle to an existing Workflow by its identifier.
+tags:
+  - developer-guide
+  - sdk
+  - python
+---
+
+Use [`get_external_workflow_handle_for`](https://python.temporal.io/temporalio.workflow.html#get_external_workflow_handle_for) to get a typed Workflow handle to an existing Workflow by its identifier.
```

Comment:
Note that `get_external_workflow_handle` is used when you don't know the type of the other workflow. These mimic the `client.get_workflow_handle` and `client.get_workflow_handle_for` methods.